op-chain-ops/go.mod

```diff
@@ -58,4 +58,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.10.21 => github.com/ethereum-optimism/op-geth v0.0.0-20220904174542-4311f9d2cead
+replace (
+	github.com/ethereum/go-ethereum v1.10.21 => github.com/ethereum-optimism/op-geth v0.0.0-20220904174542-4311f9d2cead
```

Comment:
Why are we requiring two different versions of geth here?

Reply:
Nice catch. We shouldn't have to different versions. Fixing.